includes/admin/home/tasks/class-sensei-home-tasks-provider.php

```diff
@@ -88,6 +90,50 @@ private function map_task( Sensei_Home_Task $task ): array {
 		];
 	}
 
+	/**
+	 * Return the site information needed for the task list component.
+	 *
+	 * @return array The site info, including title and image (which is the custom logo) URL.
+	 */
+	private function get_site() {
+		$custom_logo_id = get_theme_mod( 'custom_logo' );
+		$image          = wp_get_attachment_image_src( $custom_logo_id, 'full' );
+		return [
+			'title' => get_bloginfo( 'name' ),
+			'image' => $image ? $image[0] : null,
+		];
+	}
+
+	/**
+	 * Return the course information needed for the task list component, including title and image (which is the
+	 * featured image) for that course. Please note that the data for the demo course is never returned by this method.
+	 *
+	 * @return array|null The course information, including title and image URL.
```

Comment:
I think we have to update it here with the permalink too. Also in the text above.

Reply:
Good catch! Fixed in [6fd79f5](https://github.com/Automattic/sensei/pull/5936/commits/6fd79f5bb4d62f334d8091a9ecda4f486a7b98a7).